packages/viewtoggle/src/react/index.js

```diff
@@ -41,17 +57,28 @@ class Option extends React.Component {
 Option.displayName = 'ViewToggle.Option'
 Option.propTypes = {
   _i: PropTypes.number,
-  _innerRef: PropTypes.func,
   _onSelect: PropTypes.func,
-  active: PropTypes.bool
+  active: PropTypes.bool,
+  children: PropTypes.node
 }
 Option.defaultProps = {
   active: false
 }
 
-const List = glamorous.div(css['.psds-viewtoggle'])
+const List = props => (
+  <div {...styles.list(props)} {...filterReactProps(props)} />
+)
 
-const ActivePillBg = glamorous.div(css['.psds-viewtoggle__option-bg'])
+const ActivePillBg = props => {
+  return <div {...styles.activePillBg(props)} {...filterReactProps(props)} />
```

Comment:
Unless the component is separately exposed to the user, I don't think it makes sense to pass all props to it as in `{...filterReactProps(props)}`.  

In this case, I think that means that only `<ViewToggle />` and `<ViewToggle.Option />` should receive that ability.

Components that are not exposed publicly are internal details, and a user really doesn't know what to expect when sending in html-like props in those cases.

There's also the issue of duplicating props across multiple elements, as in passing `className` to internal components.

In this case, however, the main reason I wouldn't do it is that it's not needed.  The internal use of `<ActivePillBg />` and `<PillBgSpacer />` don't pass on the user-provided.  Instead, we're filtering what we as DS devs pass as props, which we should probably just trust (and verify at dev time).  It seems like unneeded work when looking at it at the moment.